x-pack/test/functional_with_es_ssl/apps/cases/group1/view_case.ts

```diff
@@ -1221,19 +1221,23 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
 
         await testSubjects.click(`case-text-custom-field-submit-button-${customFields[0].key}`);
 
+        await header.waitUntilLoadingHasFinished();
+
         await retry.waitFor('update toast exist', async () => {
           return await testSubjects.exists('toastCloseButton');
         });
 
         await testSubjects.click('toastCloseButton');
 
-        await sync.click();
+        await header.waitUntilLoadingHasFinished();
```

Comment:
We need to wait here for the case to be refreshed. Otherwise we get an 403 conflict error because the test is too fast.